src/main/java/jpa/domain/favorite/Favorite.java

```diff
@@ -1,20 +1,87 @@
 package jpa.domain.favorite;
 
 import jpa.domain.base.BaseTimeEntity;
+import jpa.domain.member.Member;
+import jpa.domain.station.Station;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
+import java.util.Objects;
 
 
 @Entity
-
 public class Favorite extends BaseTimeEntity {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "departure_station_id")
+    private Station departureStation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "arrival_station_id")
+    private Station arrivalStation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id")
+    private Member member;
+
     protected Favorite() {
     }
+
+    public Favorite(Station departureStation, Station arrivalStation) {
+        this.departureStation = departureStation;
+        this.arrivalStation = arrivalStation;
+    }
+
+    public Favorite(Station departureStation, Station arrivalStation, Member member) {
```

Comment:
```suggestion
    public Favorite(Station departureStation, Station arrivalStation, Member member) {
        this(departureStation, arrivalStation);
        this.member = member;
    }
```

1. 중복을 최대한 제거해보아요
2. departureStation, arrivalStation, member가 null을 허용해도 되나요?

Reply:
1. Favorite 뿐만 아니라 다른 entitiy 객체들도 중복제거하였습니다. 
2. NULL 불가합니다. -> 해당 건 Validate 메소드 추가하였습니다.